### PR TITLE
include 3.8.0rc1 in manylinux1

### PR DESCRIPTION
**Pull request by mattip**
Repeat gh-344 for the manylinux1 branch

### CODE REVIEW

**Comment by mattip** [comment]
The `gpg --recv-keys` command is failing:
```
gpg: requesting key 10250568 from hkp server subkeys.pgp.net
gpg: keyserver timed out
gpg: keyserver receive failed: keyserver error
```

On gh-344 the same `--recv-key` command printed

```
gpg: requesting key 10250568 from hkp server keys.gnupg.net
gpg: key 10250568: public key ... imported
gpg: no ultimately trusted keys found
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
```

How do I change the keyserver to `keys.gnupg.net`?

**Comment by mattip** [comment]
using the full key worked

File: docker/build_scripts/build_utils.sh [hunk 1]
```diff
@@ -94,6 +94,8 @@ function build_cpythons {
     # Import public keys used to verify downloaded Python source tarballs.
     # https://www.python.org/static/files/pubkeys.txt
     gpg --import ${MY_DIR}/cpython-pubkeys.txt
+    # Add version 3.8 release manager's key
```
Comment (trishankatdatadog) [inline docker/build_scripts/build_utils.sh]:
```suggestion
    # Add version 3.8 release manager's key: https://keybase.io/ambv/
```

Comment (mattip) [reply docker/build_scripts/build_utils.sh]: fixed

**Comment by mattip** [comment]
added requested flag

**Comment by mattip** [comment]
why does this sometimes succeed, sometimes fail?
```
gpg --keyserver keys.gnupg.net --recv-keys \
E3FF2839C048B25C084DEBE9B26995E310250568
```

**Comment by mattip** [comment]
That time it worked. Should we reopen again to make sure it was a once-off fluke?

**Comment by auvipy** [comment]
now its passing

**Pull request merged**